src/schema/artwork/index.ts

```diff
@@ -535,13 +535,14 @@ export const artworkFields = () => {
         ).resolve(artwork, { precision: 0 })
 
         if (domesticShipping && !internationalShipping)
-          return "Shipping: " + domesticShipping + " Continental US only"
+          return "Shipping: " + domesticShipping + " continental US only"
+        if (!domesticShipping) domesticShipping = "Free"
         return (
           "Shipping: " +
           domesticShipping +
-          " Continental US, " +
+          " continental US, " +
```

Comment:
Latest is not to capitalize random words in sentences, though `Free` seems to still be capital.

Reply:
Yea, I guess we're going sentence-case lots of places.